collator/grafana/provisioning/dashboards/host_machine.json

```diff
@@ -0,0 +1,1441 @@
+{
```

Comment:
maybe we could just point them to the node-exporter-full dashboard? That would make the guide easier.

Reply:
Merged the dashboards for simiplicity